### ISSUE TITLE
Issues connecting to Portrait

### ISSUE THREAD

**Issue by alxdncn** [Looking-Glass/blenderLKG#41]
Currently, several users are having issues connecting to the Looking Glass Portrait device via HoPS, making it impossible for them to visualize content on the device.

**Comment by atkulp** [comment]
```
> Python: Traceback (most recent call last):
>   File "C:\Users\user1\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\looking_glass_tools\looking_glass_live_view.py", line 871, in execute
>     sock = looking_glass_settings.sock
> AttributeError: module 'looking_glass_tools.looking_glass_settings' has no attribute 'sock'
> 
> location: <unknown location>:-1
```

**Comment by GottfriedHofmann** [comment]
Hello @atkulp , I see that you are using alpha0, can you try with alpha1? https://github.com/Looking-Glass/blenderLKG/releases/tag/3.0alpha1

**Comment by atkulp** [comment]
That made a great difference!  It works pretty well now.  Thanks!

**Comment by alxdncn** [comment]
This is fixed and the change will be in both version 3.0.1 and version 3.1
